content/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository.md

```diff
@@ -218,6 +218,7 @@ There are a few simple tricks to avoid committing things you don't want committe
 - Avoid the catch-all commands `git add .` and `git commit -a` on the command line—use `git add filename` and `git rm filename` to individually stage files, instead.
 - Use `git add --interactive` to individually review and stage changes within each file.
 - Use `git diff --cached` to review the changes that you have staged for commit. This is the exact diff that `git commit` will produce as long as you don't use the `-a` flag.
+- Enable push protections for your repository to detect and prevent pushes which contain hardcoded secrets. For more information, see "[AUTOTITLE](/code-security/secret-scanning/push-protection-for-repositories-and-organizations#about-push-protection-for-repositories-and-organizations)."
```

Comment:
Thanks @diberry and @jc-clark 
I work on the docs for push protection and have slightly reworded the suggestion.

The feature is called "push protection" and what's great with it, is that it prevents pushes with accidentally committed secrets from being leaked and from being committed onto the codebase ✨  

```suggestion
- Enable push protection for your repository to detect and prevent pushes which contain hardcoded secrets from being committed. For more information, see "[AUTOTITLE](/code-security/secret-scanning/push-protection-for-repositories-and-organizations#about-push-protection-for-repositories-and-organizations)."
```